modules/rostests/apitests/crt/mbtowc.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * PROJECT:         ReactOS API tests
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Tests for mbtowc
+ * COPYRIGHT:       Copyright 2020 Bișoc George <fraizeraust99 at gmail dot com>
+ */
+
+#include <apitest.h>
+
+#define WIN32_NO_STATUS
+#include <stdio.h>
+#include <stdlib.h>
+
+#define ok_wchar(x, y) ok_int(x, y)
+
+START_TEST(mbtowc)
+{
+    size_t Length;
+    wchar_t BufferDest[3];
+    char ch = 'A';
```

Comment:
A possible future improvement would be to allocate a buffer for this using `AllocateGuarded`, to show that only the first character is ever accessed.